helm/vitalam-ipfs/values.yaml

```diff
@@ -1,7 +1,27 @@
+# fullNameOverride:
 config:
-  port: 80
+  healthCheckPort: 80
+  # nodeHost: "" # This overrides vitalamNode.enabled
+  # nodePort:
+  logLevel: trace
+  ipfsApiPort: 5001
+  ipfsSwarmPort: 4001
+  ipfsDataPath: "/ipfs"
+  ipfsCommand: "/usr/local/bin/ipfs"
+  ipfsArgs:
+    - daemon
+  # ipfsBootNodeAddress: /dnsaddr/blah.test.com/p2p/PeerId # Valid Format for this is /multiaddr/PeerId see https://github.com/multiformats/multiaddr
```

Comment:
this should probably be an array

Reply:
Setting this as an array would mean doing some horrible looping in the `statefulSet` init container.  Happy to raise this as an Issue to be fixed at a later date.

Reply:
Yeah ok that's a bit of a pain. Add an issue and we'll get it in